src/index/Index.cpp

```diff
@@ -7,6 +7,7 @@
 #include "./Index.h"
 
 #include "./IndexImpl.h"
```

Comment:
While you're at it, also fix the include format above.

Reply:
Attempted in [62de9e9](https://github.com/ad-freiburg/qlever/pull/1067/commits/62de9e932b92382392eafc038fa268cb94f85253).